tests/Unit/Pypp/PyppFundamentals.hpp

```diff
@@ -212,6 +212,32 @@ struct ToPyObject<DataVector, std::nullptr_t> {
   }
 };
 
+template <>
```

Comment:
I'd recommend factoring this commit out since it'll be easier to get in and should have different reviewers

Reply:
That makes good sense :+1:. I was struggling with whether I should ask @osheamonn for this one because of this change.